CIP-0098/README.md

```diff
@@ -0,0 +1,474 @@
+  - **`arguments`** (_required_): an array with arbitrary values used as
+    arguments for the invocation of the _renderer_ (e.g.: `[123]`)
+- **`properties`** (_optional_): an object with arbitrary key/value pairs describing the token's (unique) properties
+
+### **1.b.** Argument directives
+
+Several directives for dynamic arguments can be passed to the renderer. Before rendering a DAT, viewers must resolve any directives by querying the requested data from the blockchain and passing the actual values to the renderer.
+
+_Current token_
+
+- **`@tx_hash`** (`string`): transaction hash of the mint (can for example, be used as the seed value for a Sfc32 PRNG)
+- **`@epoch`** (`number`): epoch in which the token was minted
+- **`@slot`** (`number`): slot in which the token was minted
+- **`@block`** (`number`): block in which the token was minted
+- **`@block_size`** (`number`): the size of the token's block
+- **`@block_hash`** (`string`): hash of the token's block
+- **`@owner_addresses`** (`string[]`): an array of owner addresses
+
+_Previously minted token_
+
+- **`@tx_hash.previous`** (`string | null`): transaction hash
+- **`@epoch.previous`** (`number | null`): epoch in which the token was minted
+- **`@slot.previous`** (`number | null`): slot in which the token was minted
+- **`@block.previous`** (`number | null`): block in which the token was minted
+- **`@block_size.previous`** (`number | null`): the size of the token's block
+- **`@block_hash.previous`** (`string | null`): hash of the token's block
+- **`@arguments.previous`** (`array | null`): token's _renderer_ arguments
+
+_Specific token (within the same policy_id)_
+
+- **`@tx_hash.asset_name`** (`string | null`): transaction hash
+- **`@epoch.asset_name`** (`number | null`): epoch in which the token was minted
+- **`@slot.asset_name`** (`number | null`): slot in which the token was minted
+- **`@block.asset_name`** (`number | null`): block in which the token was minted
+- **`@block_size.asset_name`** (`number | null`): the size of the token's block
+- **`@block_hash.asset_name`** (`string | null`): hash of the token's block
+- **`@arguments.asset_name`** (`array | null`): token's _renderer_ arguments
+
+_Current blockchain state_
+
+- **`@current_epoch`** (`number`): current (latest) epoch
+- **`@current_slot`** (`number`): current (latest) slot
+- **`@current_block`** (`number`): current (latest) minted block
+- **`@current_block_size`** (`number`): the size of the current block
+- **`@current_block_hash`** (`string`): hash of the current block
+
+Passing argument directives to the _renderer_ works just like static arguments. For example:
+
+```
+[
+  123,
+  "@tx_hash",
+  "@block",
+  "@current_block",
+  "@epoch.previous",
+  "@arguments.the_perfect_nft_000"
+]
+```
+
+Referencing another token's arguments does not work recursively.
+
+**Note**: The list of directives mentioned above is not final. Proposals for new directives will be considered and added over time.
```

Comment:
What would be the process for considering new proposed amendments or updates to this specification?

Reply:
I'll come back on this in reply to your main comment at the top.